Data/repoPage.data.ts

```diff
@@ -2,9 +2,11 @@ export const repoPageData =
 {
     newRepoUrl: "/new",
     newRepoName: 'testingRepo',
+    renameRepoName: 'renameTestingRepo',
     descrption: 'testingDescrption',
     deletePageUrl: '/settings',
     deleteButtonText: 'Delete this repository',
     deleteLabelTest:'I understand the consequences, delete this repository',
-    yourProfileText: '[data-ga-click="Header, go to profile, text:your profile"]'
+    yourProfileText: '[data-ga-click="Header, go to profile, text:your profile"]',
```

Comment:
please use ^ or * for this selector, defining the needed text for the test (needed text).